python/whylogs/core/constraints/helpers.py

```diff
@@ -0,0 +1,252 @@
+from typing import Union
+
+from whylogs.core.configs import SummaryConfig
+from whylogs.core.metrics.metrics import Metric
+
+from .metric_constraints import MetricConstraint, MetricsSelector
+
+
+def greater_than_number(column_name: str, number: Union[float, int]) -> MetricConstraint:
+    """Minimum value of given column must be above defined number.
+
+    Parameters
+    ----------
+    column_name : str
+        Column the constraint is applied to
+    number : float
+        reference value for applying the constraint
+
+    """
+    constraint = MetricConstraint(
+        name=f"{column_name} greater than number {number}",
+        condition=lambda x: x.min >= number,
```

Comment:
Suggest defining a function for the conditions over using lambdas for readability and ability to add some checking: e.g. a trace/callstack will be easier to read when the function is named and we can check the input is not NaN for example, and then do the comparison.

Reply:
right!